episodes/chapter4.md

```diff
@@ -0,0 +1,242 @@
+---
+title: "Using UI"
```

Comment:
I would not abbreviate UI in the title, and put emphasis on that we want to manage the server. Maybe "Managing a vantage6 server via the user interface"?